frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -1233,31 +1243,58 @@ function FeatureFlagReleaseConditions({ readOnly }: FeatureFlagReadOnlyProps): J
                                 )}
                             </Row>
                             <LemonDivider className="my-3" />
+                            {!readOnly && hasNonInstantProperty(group.properties) && (
+                                <AlertMessage type="info" className="mt-3 mb-3">
+                                    These properties aren't immediately available on first page load for unidentified
+                                    persons. This feature flag requires that at least one event is sent prior to
+                                    becoming available to your product or website.{' '}
+                                    <a
+                                        href="https://posthog.com/manual/feature-flags#persisting-feature-flags-across-authentication-steps"
```

Comment:
@neilkakkar Not sure if this is the right link, feel free to replace if it's not !